src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+
+
+public class SortCommand extends Command{
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts address book via the selected parameter.\n"
+        + "Parameters: [name] OR [phone] OR [email] OR [address]\n"
+        + "Example: " + COMMAND_WORD + " name";
+
+    public static final String MESSAGE_SUCCESS = "List has been sorted by %1$s";
+
+    private String sortOption;
+    private Comparator<Person> sortComparator;
+//    private int[] test = new int[4];
+
+    public SortCommand(String sortParameter) {
+        sortOption = sortParameter;
+
+        switch (sortParameter) {
+            case ("name"):
```

Comment:
Note the [coding style requirements](https://nus-cs2103-ay1819s2.github.io/cs2103-website/coding-standards/java/intermediate.html) for switch-case statements - there shouldn't be indentations for case clauses. It would be helpful to configure your IDE accordingly.